pages/consortium.html

```diff
@@ -0,0 +1,140 @@
+<#def TITLE>The Coq Consortium</#def>
+<#include "incl/header.html">
+
+<div class="framework">
+<div class="frameworklabel">A pool of resources and a platform for exchanges</div>
+<div class="frameworkcontent">
+
+<p>The objective of the Consortium is to make the community of users feel empowered with respect to
+    the future of the Coq system, giving them opportunities to see how the system will evolve, where it is
+    going, and to take an active part in this process. On the one hand, we wish to establish <strong>transparent
+    ways of contributing financially and technically to the development of the system</strong>; on the other
+    hand we wish to <strong>make sure that the most pressing needs of the user community are addressed
+    efficiently</strong> by providing a fantastic working environment to some of the most competent developers
```

Comment:
It looks visually like too much bold to me. Maybe try using italic, or emphasizing less words?

Reply:
It looks fine to me.

Reply:
Maybe show it to @ybertot in order to get a second opinion.

Reply:
Ok, I reduced a bit the number of words in bold.